lib/ronin/support/text/text_score.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+require 'ronin/support/text/text_score/engine'
+
+module Ronin
+  module Support
+    module Text
+      #
+      # Scoring provides basic text scoring capibilites for
+      # english. Can be extended to other languages or formats
+      # using provided generator script in
+      # `ronin/support/text/text_score/`
+      #
+      # You'll need to modifiy the erb template to specify language
+      # if you plan to do this later. @todo: shellspawn
+      #
+      module Scoring
+        #
+        # @api public
+        #
+        # @param [Array<String>] data_set
+        #   the array of english text canidates
+        #
+        # @return [<String>, Score] or Array[[<String>, Score]]
+        #   Score is the result of the provided metric applied to
+        #   the string. This function returns the one that looks
+        #   most like english, or the top_n number of scores
+        #
+        def self.find_english_text(data_set, top_n=1)
```

Comment:
Consider using a keyword argument for `top_n:` to make it clear what the passed in argument is for.